Kinvey/KinveyTests/NetworkStoreTests.swift

```diff
@@ -2512,10 +2520,10 @@ class NetworkStoreTests: StoreTestCase {
         
         weak var expectationFind = expectation(description: "Find")
         
-        store.find(options: Options(readPolicy: .forceNetwork)) { (result: Result<AnyRandomAccessCollection<Products>, Swift.Error>) in
+        store.find(options: Options(readPolicy: .forceNetwork, maxSizePerResultSet: pageSizeLimit)) { (result: Result<AnyRandomAccessCollection<Products>, Swift.Error>) in
```

Comment:
I'm confused, where is the code that adds this to `Options`? What is the relation between `Options` and `RequestFactory`?

Reply:
the `Options` struct is in the same file of `RequestFactory`. We can create a separate file for `Options` as well. I will do that.